rdt/transformers/CatTransformer.py

```diff
@@ -20,9 +20,8 @@ def fit_transform(self, col, col_meta, missing=True):
         """ Returns a tuple (transformed_table, new_table_meta) """
         out = pd.DataFrame()
         col_name = col_meta['name']
-        # Make sure all nans are handled the same by replacing
-        # with -1
-        column = col.fillna(-1)
+        # Make sure all nans are handled the same by replacing with None
+        column = col.replace({np.nan: None})
```

Comment:
This line shouldn't be here. With the replacement inside `get_probability_map` should be enough.

Reply:
It does not work without it